test/e2e/rules_test.go

```diff
@@ -0,0 +1,170 @@
+// +build integration
+
+package e2e
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/efficientgo/e2e"
+	"github.com/efficientgo/tools/core/pkg/testutil"
+)
+
+func TestRulesAPI(t *testing.T) {
+	t.Parallel()
+
+	e, err := e2e.NewDockerEnvironment(envRulesAPIName)
+	testutil.Ok(t, err)
+	t.Cleanup(e.Close)
+
+	prepareConfigsAndCerts(t, rules, e)
+	_, token, rateLimiterAddr := startBaseServices(t, e, rules)
+	rulesEndpoint := startServicesForRules(t, e)
+
+	api, err := newObservatoriumAPIService(
+		e,
+		withRulesEndpoint("http://"+rulesEndpoint),
+		withRateLimiter(rateLimiterAddr),
+	)
+	testutil.Ok(t, err)
+	testutil.Ok(t, e2e.StartAndWaitReady(api))
+
+	rulesEndpointURL := "https://" + api.Endpoint("https") + "/api/metrics/v1/test-oidc/api/v1/rules/raw"
```

Comment:
Is the `test-oidc` tenant name defined anywhere else? I just imagine if we change that we'll unexpectedly break this test

Reply:
good point..we define it manually everywhere in the tests: https://github.com/observatorium/api/search?q=test-oidc 
maybe makes sense to define it as a constant in `configs.go` and use its value? I could open another pr for that or just already change it here (?)

Reply:
added as a constant here: https://github.com/observatorium/api/pull/198/commits/0b6000559c7d58f91e53a48627eef6fda60f87dc